doc/nrf/known_issues.rst

```diff
@@ -577,7 +590,7 @@ KRKNWK-12017: Zigbee End Device doesn't recover from broken rejoin procedure
 KRKNWK-8211: Leave signal generated twice
   The ``ZB_ZDO_SIGNAL_LEAVE`` signal is generated twice during Zigbee Coordinator factory reset.
 
-.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+.. rst-class:: v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
 
 KRKNWK-9714: Device association fails if the Request Key packet is retransmitted
   If the Request Key packet for the TCLK is retransmitted and the coordinator sends two new keys that are different, a joiner logic error happens that leads to unsuccessful key verification.
```

Comment:
This one is missing from the following list: https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+1.9.0

Which suggests it might've been fixed. @maciejpietras to confirm.

Reply:
Fixed

Reply:
Removed the 1-9-0 tag.